src/toolbars/ToolBarButtons.h

```diff
@@ -0,0 +1,65 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  ToolBarButtons.h
+
+  ksoze95
+
+**********************************************************************/
+
+#ifndef __AUDACITY_TOOLBAR_BUTTONS__
+#define __AUDACITY_TOOLBAR_BUTTONS__
+
+#include <functional>
+#include <vector>
+#include "ToolBar.h"
+
+class wxCommandEvent;
+
+class AudacityProject;
+class AButton;
+
+struct Entry {
```

Comment:
I don't like a simple bland name like `Entry` in the global namespace.  This might be a nested type in `ToolBarButtons`.